src/analyzer/analyzerwaveform.h

```diff
@@ -81,15 +74,23 @@ struct WaveformStride {
                     m_postScaleConversion * scaleSignal(m_filteredData[i][Mid], Mid) + 0.5));
             datum.filtered.high = static_cast<unsigned char>(math_min(255.0,
                     m_postScaleConversion * scaleSignal(m_filteredData[i][High], High) + 0.5));
+            for (int stemIdx = 0; stemIdx < m_stemCount; stemIdx++) {
+                datum.stems[stemIdx] = static_cast<unsigned char>(math_min(255.0,
+                        m_postScaleConversion * scaleSignal(m_stemData[i][stemIdx]) + 0.5));
```

Comment:
scaleSignal() does nothing here and is not prepared allow it later. Remove?

Reply:
I must say I'm not entirely up to speed of what scaleSignal is supposed to do, only reused the same logic as above. Will remove and we can see if that changes anything 
@ywwg I remember during one of our catch up, you mentioned some legacy hardcoded scaling on the waveform signal samples that should be removed, is that it?

Reply:
It currently just scales the bass stream. We can't change it, because the scale factor goes to the stored waveform data.

Reply:
Than scaleSignal is only relevant for the compatibility to existing waveform cache files, but for stem waveforms we can do it right from the begining and don't apply any scaling in the analyzer.

Reply:
I think we should take the opportunity to rip this out, once and for all.  If we are bumping the analysis version number then we can remove this for all analysis and fix the waveform widgets appropriately

Reply:
(the original reason I wrote the scaling was because it was hard to see the bass signal in the waveforms)

Reply:
Great! Yes, I also think it should be ripped off because of the bump off. I will put it to my list and get it done in a follow up PR tho just to separate it from the stem, since I assume it will also need some logic change in the renderer, where there is is that `x2` logic.

Reply:
For this PR, please just remove the scaleSignal() call from the new code. 

I am a bit in doubt whether the version bump is worth the effort to get rid of this. We need also consider that this makes it harder to go back an forth between release and beta. We just need to make sure to have it on the to-do lost if we need to make a version bump anyway.

Reply:
> I am a bit in doubt whether the version bump is worth the effort to get rid of this.

We need to as currently, the render applies a gain on sample due to this, so if we fix this, old waveform will look unusual due to missing gain. 
Since the stem feature will require a version bump, this is why I suggested making the signal clean up as well, so we can do a "2 birds one stone".

Reply:
> I think we should take the opportunity to rip this out, once and for all. If we are bumping the analysis version number then we can remove this for all analysis and fix the waveform widgets appropriately

Going to try an prepare a PR today

Reply:
#13416